com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -260,16 +335,60 @@ public enum Authority
         private AnimatorSnapshot m_AnimatorSnapshot;
         private List<(int, AnimatorControllerParameterType)> m_CachedAnimatorParameters;
 
-        public bool IsAuthorityOverAnimator => (IsClient && AnimatorAuthority == Authority.Owner && IsOwner) || (IsServer && AnimatorAuthority == Authority.Server);
+        private ulong[] m_ServerMessagingTargetClientIds;
+        private Dictionary<ulong, ulong[]> m_TargetClientIdsCache;
+
+        private bool m_Initialized = false;
+
+        /// <summary>
+        /// This property tells us if the changes made to the Mecanim Animator will be synced to other peers or not.
+        /// If not - then whatever local changes are done to the Mecanim Animator - they'll get overriden.
+        /// </summary>
+        public virtual bool WillCommitChanges => IsServer;
```

Comment:
Any reason why we use `Will` here compared to `CanCommit` of the NetworkTransform. I think it would be nice if we could have a consistent API here?

Reply:
I don't mind changing it to CanCommit.

Reply:
CanCommitToAnimator - to match the pattern from NetworkTransform